src/python/example/diagram_vectorizations_distances_kernels.py

```diff
@@ -102,7 +118,9 @@ def arctan(C,p):
 Y = PSS.transform(diags2)
 print("PSS kernel is " + str(Y[0][0]))
 
-PSS = PersistenceScaleSpaceKernel(kernel_approx=RBFSampler(gamma=1./2, n_components=100000).fit(np.ones([1,2])))
+PSS = PersistenceScaleSpaceKernel(
+    kernel_approx=RBFSampler(gamma=1. /
+                             2, n_components=100000).fit(np.ones([1, 2])))
```

Comment:
That seems like a strange place to break, or even to add spaces. Also, why replace 1. with 1.0 in Wasserstein but not here?

Reply:
This should have been `gamma=.5` anyway, and then both black and yapf (all 4 default styles) give something vaguely sensible.